tests/Tests/600__ai_apps/660__openvino/660__openvino.robot

```diff
@@ -12,37 +12,47 @@ Suite Teardown  OpenVino Suite Teardown
 *** Variables ***
 ${openvino_appname}           ovms
 ${openvino_container_name}    OpenVINO
-${openvino_operator_name}    OpenVINO Toolkit Operator
+${openvino_operator_name}     OpenVINO Toolkit Operator
+${openvino_build_label}       opendatahub.io/build_type\n=\nnotebook_image
 
 *** Test Cases ***
 Verify OpenVino Is Available In RHODS Dashboard Explore Page
-  [Tags]  ODS-258  Smoke  Sanity
-  Open Browser  ${ODH_DASHBOARD_URL}  browser=${BROWSER.NAME}  options=${BROWSER.OPTIONS}
-  Login To RHODS Dashboard  ${TEST_USER.USERNAME}  ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
-  Wait for RHODS Dashboard to Load
-  Verify Service Is Available In The Explore Page    OpenVINO
-  Verify Service Provides "Get Started" Button In The Explore Page    OpenVINO
+   [Tags]  ODS-258  Smoke  Sanity
+   Open Browser  ${ODH_DASHBOARD_URL}  browser=${BROWSER.NAME}  options=${BROWSER.OPTIONS}
+   Login To RHODS Dashboard  ${TEST_USER.USERNAME}  ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
+   Wait for RHODS Dashboard to Load
+   Verify Service Is Available In The Explore Page    OpenVINO
+   Verify Service Provides "Get Started" Button In The Explore Page    OpenVINO
 
 Verify Openvino Operator Can Be Installed Using OpenShift Console
    [Tags]   Tier2
    ...      ODS-675
+   ...      ODS-652
    ...      ODS-495
    ...      ODS-1236
-   [Documentation]  This Test Case Installed Openvino operator in Openshift cluster
+   [Documentation]  This Test Case Installed Openvino operator in Openshift cluster,
+   ...              Cheks whether RHODS notification shows the Notebook images are running after installing openvino
    ...               and Check and Launch AIKIT notebook image from RHODS dashboard
-   Check And Install Operator in Openshift    ${openvino_operator_name}   ${openvino_appname}
+   Run Keyword and Ignore Error    Check And Install Operator in Openshift    ${openvino_operator_name}   ${openvino_appname}
    Create Tabname Instance For Installed Operator        ${openvino_operator_name}       Notebook    redhat-ods-applications
    Wait Until Keyword Succeeds    900  1     Check Image Build Status   Complete     openvino-notebook
+   Openvino Build Should Contain The Label    ${openvino_build_label}
    Go To RHODS Dashboard
+   RHODS Notification Drawer Should Contain    message=Notebook images are building
    Verify Service Is Enabled          ${openvino_container_name}
    Verify JupyterHub Can Spawn Openvino Notebook
    [Teardown]   Uninstall Openvino Operator
 
-
 *** Keywords ***
+Openvino Build Should Contain The Label
```

Comment:
You could replace this keyword for a new keyword in tests/Resources/Page/OCPDashboard/Builds/Builds.robot:
```
Build Should Contain Label      ${namespace}    ${build_search_term}    ${required_label]
```

Reply:
Done

Reply:
I don't see it in Builds.robot